<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_src_open">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>open source image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tar_open">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>open target image</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="tree_src_point">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>141</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>select 4 points</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QTreeWidget" name="tree_tar_point">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>141</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>select 4 points</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_src_del">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_src_clear">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>clear all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tar_del">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tar_clear">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>clear all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_perspective">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Perspective Transform</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_undo">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo Transform</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
